content/issues/2022-01-13-airsonic.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Airsonic Deprecation Notice
+date: 2022-01-13 18:00:00Z
+informational: true
+affected:
+  - Deprecations
+  - airsonic
+section: issue
+---
+
+Due to [ending of support](https://github.com/airsonic/airsonic#airsonic-isnt-maintained-anymore-you-should-migrate-to-airsonic-advanced-instead) by the upstream project, we have made the decision to deprecate our Airsonic image. The image will immediately cease to be supported and will not receive any further updates.
+
+Users of the image are advised to switch to our [Airsonic-Advanced](https://github.com/airsonic-advanced/airsonic-advanced) image instead.
```

Comment:
Should it link to our image somewhere?

Reply:
You mean the existing one? Yeah probably.